addons/website_sale_delivery/controllers/main.py

```diff
@@ -76,3 +93,9 @@ def _update_website_sale_delivery_return(self, order, **post):
                     'new_amount_total': self._format_amount(order.amount_total, currency),
             }
         return {}
+
+    def _format_amount(self, amount, currency):
```

Comment:
to replace

self.env['ir.qweb.field.monetary'].value_to_html

Reply:
:')